dotcom-rendering/src/components/ArticleMeta.apps.tsx

```diff
@@ -105,7 +105,7 @@ const MetaGridByline = ({
 				padding-bottom: 0px;
 			}
 
-			address > div > span > svg:first-of-type {
+			address > div > span > div:first-of-type {
```

Comment:
The `DottedLines` component in Source Development Kitchen has been updated to be rendered with CSS rather than as an SVG, therefore there's no longer an `svg` element to target here. This selector feels very fragile and the spacing would be better applied elsewhere, but I've left as is for now and simply updated the element it's targeting.